modules/openapi-generator/src/main/resources/rust-server/server-operation.mustache

```diff
@@ -319,13 +319,48 @@
                                                 return Ok(
                                                     Response::builder()
                                                     .status(StatusCode::BAD_REQUEST)
-                                                    .body(Body::from(format!("{{{paramName}}} data does not match API definition : {}", e)))
-                                                    .expect("Unable to create Bad Request due to missing required form parameter {{{paramName}}}"))
+                                                    .body(Body::from(format!("{{{baseName}}} data does not match API definition : {}", e)))
+                                                    .expect("Unable to create Bad Request due to missing required form parameter {{{baseName}}}"))
                                             }
                                         };
                                         {{{paramName}}}_model
                                   {{/jsonSchema}}
                                 {{/isByteArray}}
+                                {{#isModel}}
```

Comment:
mustache not support OR syntax like `{{#isByteArray || #isModel}}`.
any idea to avoid this copy-paste (from the above L312-L328)?

Reply:
What about the following?
```
{{#isByteArray}}
...
{{/isByteArray}}
{{^isByteArray}}
{{#isModel}}
...
{{/isModel}}
{{/isByteArray}}
```

Reply:
sorry, this PR still had another bug.

(pseudo code)

```
if `format: binary` {
  return `load $param as [u8]`
}

if `format: byte` {
  $x = `decode $param as JSON string`  // ← this causes an error: binary_field data does not match API definition : expected value at line 1 column 1

  return `decode $x as base64`
}

if `model` {
  return `load $param by serde_json::from_str`
}

if `primitive type (ex. int64)` {
  return `load $param by FromStr<int64>`
}
```

I've fixed this error like below :

```
if `format: binary` {
  return `load $param as [u8]`
}

if `model` {
  return `load $param by serde_json::from_str`
}

if `primitive type (ex. ByteArray)` {
  // (ByteArray::from_str can decode base64 string.)
  return `load $param by FromStr<ByteArray>`
}
```

copy-paste is no longer necessary. thanks.